generic/htx_test.py.data/htx_test.yaml

```diff
@@ -0,0 +1,3 @@
+time_limit: 2
```

Comment:
You should remove the `generic/htx_test.data/htx_test.yaml `

Reply:
oh sorry forgot that will do , thanks for review